src/components/dynamic/admin/dashboards/dashboard/Modal.jsx

```diff
@@ -11,7 +12,12 @@ const Modal = ({ data, setModal }) => {
         />
       </div>
       <div className="bg-white px-3 rounded-b-xl py-2 border-x-2 border-b-2 border-hackathon-darkgray h-full w-full flex justify-center">
-        <img className="h-full" src={data.src} />
+        <Image
+          fill={true}
+          className="h-full"
+          src={data.src}
+          alt="user image here"
```

Comment:
better alt text

Reply:
I noticed that its only being used to display the image for the judges (Modal's wrapper, View, is also used in participants as well but I don't see images being displayed there), so I think I'll put "Photo of the Judge" instead,